packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx

```diff
@@ -1,35 +1,139 @@
+          position="absolute"
+        >
+          <FooterBuildingBlocks aria-hidden="true" role="img" />
+        </Box>
+        <Box position="relative">
+          <Box
+            alignItems="center"
+            display="flex"
+            flexDirection={['column', 'row']}
+            marginBottom="space80"
+            textAlign={['center', 'left']}
+          >
+            <HeartDoodleIcon
+              css={{height: theme.heights.sizeIcon70, width: theme.widths.sizeIcon70}}
+              color={theme.textColors.colorTextInverse}
+              decorative
+            />
+            <Text
+              as="h5"
+              color="colorTextInverse"
+              fontSize={['fontSize60', 'fontSize90']}
+              fontWeight="fontWeightBold"
+              lineHeight={['lineHeight60', 'lineHeight90']}
+              marginLeft={['space0', 'space40']}
+              marginTop={['space40', 'space0']}
+            >
+              Black lives matter.
+            </Text>
+          </Box>
+          <Box
+            display="flex"
+            flexDirection={['column', 'row']}
+            marginLeft={['space0', 'space120']}
+            textAlign={['center', 'left']}
+          >
+            <Box marginRight={['space0', 'space130']} marginBottom="space60">
+              <Stack as="nav" orientation="vertical" spacing="space60">
+                <Anchor href="/about" variant="inverse">
+                  About
+                </Anchor>
+                <Anchor href="/roadmap" variant="inverse">
+                  Our roadmap
+                </Anchor>
+                <Anchor href="/getting-started/design" variant="inverse">
+                  Design guidelines
+                </Anchor>
+                <Anchor href="/getting-started/engineering" variant="inverse">
+                  Engineering guidelines
+                </Anchor>
+              </Stack>
+            </Box>
+            <Box marginRight={['space0', 'space130']} marginBottom="space60">
+              <Stack as="nav" orientation="vertical" spacing="space60">
+                <Anchor href="https://github.com/twilio-labs/paste/discussions" variant="inverse" showExternal>
+                  Get support
+                </Anchor>
+                <Anchor href="https://github.com/twilio-labs/paste/issues/new/choose" variant="inverse" showExternal>
+                  Report a bug
+                </Anchor>
+                <Anchor href="/how-we-work" variant="inverse">
+                  How we work
+                </Anchor>
+              </Stack>
+            </Box>
+            <Box marginRight={['space0', 'space130']} marginBottom="space60">
+              <Stack as="nav" orientation="vertical" spacing="space60">
+                <Anchor href="#" variant="inverse" showExternal>
+                  Abstract
+                </Anchor>
+                <Anchor href="#" variant="inverse" showExternal>
+                  Figma
+                </Anchor>
```

Comment:
This'll be removed before we ship, yeah? Or we could make it static text that says "Figma (coming soon)".

Reply:
We can remove it for now.